releasetasks/util.py

```diff
@@ -0,0 +1,22 @@
+import os
+from taskcluster.utils import encryptEnvVar
+
+
+def treeherder_platform(platform):
+    # See https://github.com/mozilla/treeherder/blob/master/ui/js/values.js
+    m = {
+        "linux": "linux32",
+        "linux64": "linux64",
+        "macosx64": "osx-10-10",
+        "win32": "windowsxp",
+        "win64": "windows8-64",
+    }
+    return m[platform]
```

Comment:
Can you ask now, at least? If it's a simple cut and paste into thclient it saves us the trouble of maintaining it for now...and we have a bad habit of not coming back around to things like this.

If it's more than a trivial amount of work we can just keep it here for now.